docs/docs/KerasStyleAPIGuide/Layers/core.md

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
Delete this line?

Reply:
@jason-dai By adding this, there will be a segmentation line between two layers, which will make the layer look nicer.
See https://bigdl-project.github.io/master/#ScalaUserGuide/install-pre-built/ (with ---)
and https://bigdl-project.github.io/master/#ScalaUserGuide/install-build-src/ (without ---) for comparison. The former should look clearer.